<launch>
  <arg name="group" default="A" />
  <arg name="drone_num" default="3" />
  <arg name="rosbag_id" default="15" />
  <arg name="drone_data" default="[]" />
  <group ns="$(arg group)">

    <!-- Start the swarm_driver -->
    <node name="hardcode_swarm_driver_node" pkg="drone_swarm" type="hardcode_swarm_driver.py" output="screen">
      <param name="~group" value="$(arg group)"/>
      <param name="~drone_num" value="$(arg drone_num)"/>
      <param name="~rosbag_id" value="$(arg rosbag_id)"/>
      <param name="~drone_data" value="$(arg drone_data)"/>
    </node>

  </group>

</launch>
